message/message.yml

```diff
@@ -0,0 +1,412 @@
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v1/sessions/{token}/subscriptions/subscribe:
+    parameters:
+      - $ref: '#/parameters/Token'
+    post:
+      tags:
+        - Topics
+      summary: Subscribe to a topic
+      description: Subscribes to a topic to receive all messages broadcast with that topic name.
+      operationId: SubscribeToTopic
+      responses:
+        200:
+          description: Success
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+      parameters:
+        - in: body
+          name: topic
+          description: The topic to subscribe to
+          required: true
+          schema:
+            $ref: '#/definitions/Topic'
+  /v1/sessions/{token}/subscriptions/unsubscribe:
+    parameters:
+      - $ref: '#/parameters/Token'
+    post:
+      tags:
+        - Topics
+      summary: Unsubscribe from a topic
+      description: Unsubscribes from a topic to stop receiving messages for that topic.
+      operationId: UnsubscribeFromTopic
+      responses:
+        200:
+          description: Success
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+      parameters:
+        - in: body
+          name: topic
+          description: The topic to unsubscribe from
+          required: true
+          schema:
+            $ref: '#/definitions/Topic'
+
+  /v1/sessions/{token}/messages:
+    parameters:
+      - $ref: '#/parameters/Token'
+    post:
+      tags:
+        - Message Queue
+      summary: Publish a message
+      description: Publishes a broadcast message to a specific topic. All of the topic's subscribers will receive a copy of the message.
+      operationId: PublishMessage
+      responses:
+        200:
+          description: Success
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+      parameters:
+        - in: body
+          name: topicAndMessage
+          description: The message to publish, and the topic to publish to
+          required: true
+          schema:
+            $ref: '#/definitions/TopicAndMessage'
+    get:
+      tags:
+        - Message Queue
+      summary: Read a message from the queue
+      description: Read a single message from the queue, if any.
+      operationId: ReadMessage
+      responses:
+        200:
+          schema:
+            $ref: '#/definitions/TopicAndMessage'
+          description: Success
+        204:
+          description: No messages were available.
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+      parameters:
+        - in: query
+          name: timeoutMilliseconds
+          description: The time to keep the connection open while waiting for a message
+          type: integer
+          required: false
```

Comment:
Do we know if the default timeout is consistent between server and cloud? If it is, we could document it if we wanted to. We may also want to mention in the description that there's a maximum (again, we can check if there's a consistent max or not).

Reply:
From testing, they both seem to have a max timeout of ten seconds. Server seems to have a default timeout of one second, but I'm not sure what Cloud's is. I'll document these values once we know for sure.

@tschmittni: do you know what SystemLink Cloud uses as the default value for timeoutMilliseconds?